pipeline/download_resources_files.sh

```diff
@@ -10,16 +10,38 @@ echo "Downloading into $1 ...."
 
 cd $1
 
+URL_PREFIX="http://brcaexchange.org/backend/downloads/resources"
 
-wget http://hgwdev.soe.ucsc.edu/~cline/BRCA/resources/refseq_annotation.hg38.gp
-wget http://hgwdev.soe.ucsc.edu/~cline/BRCA/resources/refseq_annotation.hg19.gp
-wget http://hgwdev.soe.ucsc.edu/~cline/BRCA/resources/refseq_annotation.hg18.gp
-wget http://hgwdev.soe.ucsc.edu/~cline/BRCA/resources/hg19.fa.gz && gunzip hg19.fa.gz
-wget http://hgwdev.soe.ucsc.edu/~cline/BRCA/resources/hg18.fa.gz && gunzip hg18.fa.gz
-wget http://hgdownload.cse.ucsc.edu/goldenPath/hg38/bigZips/hg38.fa.gz && gunzip hg38.fa.gz
-wget http://hgdownload.cse.ucsc.edu/gbdb/hg19/liftOver/hg19ToHg38.over.chain.gz
-wget http://hgwdev.soe.ucsc.edu/~cline/BRCA/resources/brca1_hg38.txt
-wget http://hgwdev.soe.ucsc.edu/~cline/BRCA/resources/brca2_hg38.txt
-wget http://hgwdev.soe.ucsc.edu/~cline/BRCA/resources/brca1_hg19.txt
-wget http://hgwdev.soe.ucsc.edu/~cline/BRCA/resources/brca2_hg19.txt
+HASH=shasum
 
+declare -A files
+
+files[brca1_hg19.txt]=0103381d9f931367e85cc7d4c0504973a6ef08e2
+files[brca1_hg38.txt]=1adac34fa794b8057926b836384c52fe53934b6a
+files[brca2_hg19.txt]=a0f76d6a61d5275be4695e9628b0bf8d49dc1a37
+files[brca2_hg38.txt]=e3d40430167100bb3252a5f0e3655f56e244e413
+files[hg18.fa.gz]=177dcd8262eed25f565cd2cd10f87fe63a7e54cb
+files[hg19.fa.gz]=c1a06a3122bb9b3747ceeefb902aecd3ea9b9008
+files[hg19ToHg38.over.chain.gz]=4769aaa7a5a7837c3947604207da346b1dd19a8f
+files[hg38.fa.gz]=d329507d63d32405664053785b2bfdc02b3e6b28
+files[refseq_annotation.hg18.gp]=21911f9cd02f00b82eeb6068b7ab737c4d89f896
+files[refseq_annotation.hg19.gp]=e12092220926fe112da26ffdc8188b76aef83231
+files[refseq_annotation.hg38.gp]=9fb29159e516d2f28c2ec0e41a338c85ac699528
+
+for file in "${!files[@]}"; do
+    hash=$(${HASH} $file | cut -f1 -d' ')
+    if [ "$hash" != "${files[$file]}" ]; then
```

Comment:
should this be `==` instead of `!=` ?

Reply:
no, I think it's fine. In case the file already exists but the hash doesn't match it gets redownloaded